doc/api/crypto.md

```diff
@@ -5968,7 +5968,7 @@ See the [list of SSL OP Flags][] for details.
   </tr>
   <tr>
     <td><code>ENGINE_METHOD_PKEY_METHS</code></td>
-    <td>Limit engine usage to PKEY_METHDS</td>
```

Comment:
this is not a typo per se, it comes from an upstream dependency

Reply:
@panva this is not aligned with rest of the file though:

- https://github.com/nodejs/node/pull/55066/files#diff-52a58775d8a5519e232893c0d176d1a7461037007ac2c8f25a2aa803b8304454R5326
- https://github.com/nodejs/node/pull/55066/files#diff-52a58775d8a5519e232893c0d176d1a7461037007ac2c8f25a2aa803b8304454R5974
-